distributions/raspijamulus.sh

```diff
@@ -51,7 +51,7 @@ else
     patch celt/celt_decoder.c opus_patch_file.diff
   fi
   ./configure --enable-custom-modes --enable-fixed-point
-  make -j${NCORES}
+  make -j"${NCORES}"
```

Comment:
And `${NCORES}` is just a number. Again, quoting is unnecessary and cumbersome.

Reply:
I wonder if Codacy understands `readonly` and then would work out that the expression is safe?  Somehow I doubt it.

Reply:
Using `declare -ri NCORES=(whatever)` might allow Codacy to realise it's an integer constant, too.